cumulusci/tasks/github/tests/test_merge.py

```diff
@@ -192,9 +202,11 @@ def test_no_feature_branch(self):
             expected = [
                 ('INFO', 'Beginning task: MergeBranch'),
                 ('INFO', ''),
+                ('DEBUG', 'Skipping branch master: is source branch'),
+                ('DEBUG', 'Skipping branch not-a-feature-branch: does not match prefix feature/'),
             ]
-            self.assertEquals(log_lines, expected)
-        self.assertEquals(len(responses.calls), 4)
+            self.assertEquals(expected, log_lines)
```

Comment:
is there an easy way to just assert that each of the expected lines is IN the log? i don't like that adding additional logging could break the test, but I don't feel that strongly abt this.

Reply:
I kind of like that it's an all or nothing thing.  That allows us to test that content is not shown in the log as well as that it is.